ci_images/focal_builder.dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apt update -q \
   && pip3 install meson pytest gcovr requests distro
 
 # Create user
-RUN useradd --create-home runner
+RUN groupadd --gid 121 runner
+RUN useradd --uid 1001 --gid 121 --create-home runner
 USER runner
-WORKDIR /home/runner
```

Comment:
This (and other change in the same commit) remove the WORKDIR setting.
Is it voluntary ? Why we don't need it ?

Reply:
"GitHub sets the working directory path in the GITHUB_WORKSPACE environment variable. It's recommended to not use the WORKDIR instruction in your Dockerfile." https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#workdir